client/my-sites/people/people-list-item/index.jsx

```diff
@@ -42,29 +45,41 @@ export default React.createClass( {
 			this.userHasPromoteCapability() &&
 			! this.props.isSelectable
 		);
-	},
+	};
 
 	render() {
 		const canLinkToProfile = this.canLinkToProfile();
+
 		return (
 			<CompactCard
-				{ ...omit( this.props, 'className', 'user', 'site', 'isSelectable', 'onRemove' ) }
+				type={ this.props.type }
 				className={ classNames( 'people-list-item', this.props.className ) }
 				tagName="a"
 				href={ canLinkToProfile && '/people/edit/' + this.props.site.slug + '/' + this.props.user.login }
-				onClick={ canLinkToProfile && this.navigateToUser }>
+				onClick={ canLinkToProfile && this.props.navigateToUser }>
 				<div className="people-list-item__profile-container">
 					<PeopleProfile user={ this.props.user } />
 				</div>
 				{
 				this.props.onRemove &&
 				<div className="people-list-item__actions">
 					<button className="button is-link people-list-item__remove-button" onClick={ this.props.onRemove }>
-						{ this.translate( 'Remove', { context: 'Verb: Remove a user or follower from the blog.' } ) }
+						{ this.props.translate( 'Remove' ) }
 					</button>
 				</div>
 				}
 			</CompactCard>
 		);
 	}
-} );
+}
+
+const navigateToUser = () => {
+	browserWindow.scrollTo( 0, 0 );
```

Comment:
Sounds great @dmsnell although if I remove the `scrollTo( 0, 0 )` entirely, I see no visible change in behaviour.

The a href causes a route change which makes the scrolling redundant. 

I'm considering just removing it.